contents/handbook/growth/marketing/hosthogs.md

```diff
@@ -0,0 +1,56 @@
+---
+title: HostHogs
+sidebar: Handbook
+showTitle: true
+---
+
+HostHogs are our global series of meetups. The mission of HostHogs is to support and grow the PostHog community through targeted in-person events that offer an opportunity for like-minded members and potential members of the community to meet and benefit from deeper conversations than are usually possible in our existing GitHub and Slack communities. 
+
+Specific product metrics that this supports (in order):
+1. Retention
+2. Referral
+3. Awareness
+
+## Target audience
+
+We think HostHogs will be of particular interest to people in the following roles at our [target companies](/handbook/strategy/strategy):
+
+- Product Engineer
+- Senior/Lead Engineer
```

Comment:
Technical decision makers as well maybe?

Reply:
I'd keep it to just specific roles, I think technical decision makers isn't specific enough to me

Reply:
Should have been more specific, I was thinking of tech leads

Reply:
Ah yep, agreed - added